inspector/src/tabs/TextureTab.ts

```diff
@@ -196,4 +227,101 @@ export class TextureTab extends Tab {
         }
         item.active(true);
     }
+
+}
+
+class TestDDSPreview {

```

Comment:
Not fan of having test class here

Reply:
Yes you are right and right, i'm fixing that asap.

I was not aware of the webGL context limit, I need to check that.